components/opponent/commons/player_ext.lua

```diff
@@ -311,6 +320,9 @@ function PlayerExt.syncTeam(pageName, template, options)
 		playerVars:set(pageName .. '.teamHistory', Json.stringify(history))
 	end
 
+	if options.returnRaw then
+		return entry and entry.raw or nil
+	end
 	return entry and entry.template or nil
```

Comment:
Could do a double return instead, I'll leave it up to you which one you think is better

```suggestion
	if not entry then
		return nil
	end
	return entry.template, entry.raw
```

Reply:
I'm writing too much Python where double returns work differently, so didn't consider this.
IMO it's a lot cleaner to not require the flag